nuxeo-core-binarymanager-s3/src/main/java/org/nuxeo/ecm/core/storage/sql/S3BinaryManager.java

```diff
@@ -224,7 +229,8 @@ protected void setupCloudClient() throws IOException {
         String endpoint = getProperty(ENDPOINT_PROPERTY);
         String sseprop = getProperty(SERVERSIDE_ENCRYPTION_PROPERTY);
         if (isNotBlank(sseprop)) {
-            userServerSideEncryption = Boolean.parseBoolean(sseprop);
```

Comment:
Can you put this into a separate commit, like `Fix Typo` ?

Reply:
Ideally yes but it's a small enough fix that this doesn't matter here.